frontend/www/js/omegaup/components/qualitynomination/ReviewerPopup.vue

```diff
@@ -62,7 +66,7 @@
               </th>
             </thead>
             <tbody>
-              <tr v-for="tag in publicTagsList">
+              <tr v-for="(tag, index) in publicTagsList" :key="index">
```

Comment:
```suggestion
              <tr v-for="(tag, index) in publicTagsList" :key="tag">
```

en un `v-for`, se recomienda no usar el índice como llave: si se cambian los contenidos del elemento del arreglo, Vue no tiene cómo darse cuenta de algunos de esos cambios: https://forum.vuejs.org/t/v-for-with-simple-arrays-what-key-to-use/13692/11

Reply:
Listo.